atoms.js

```diff
@@ -37,25 +37,25 @@ function AtomRW(width, readAtomFrom, writeAtomInto) {
 }
 inherits(AtomRW, BufferRW);
 
-AtomRW.prototype.byteLength = function byteLength() {
-    return new LengthResult(null, this.width);
+AtomRW.prototype.poolByteLength = function byteLength(destResult) {
```

Comment:
No, it inherits from BufferRW, which provides the compatibility layer. The `BufferRW#byteLength` method will allocate the result object and then call `poolByteLength`.